docs/wallet_API.md

```diff
@@ -6,27 +6,27 @@ author: Claude Barde
 
 ## What is the Wallet API?
 
-The Tezos blockchain is a fantastic tool, but it is self-contained. Except for the transactions you send to it, it has no interaction with the outside world. However, it would be amazing to interact with it, such as making payments, reading the balance of an account, or recording data in a smart contract. This interaction is what the Wallet API (and Taquito in general) achieves. The Wallet API provides a new yet familiar way to interact with the blockchain and smart contracts by delegating several actions that Taquito previously handled to the wallets. This delegation offers more flexibility for both developers and users and gives the ecosystem more space to evolve.
+You have learned how to use Taquito to interact with the Tezos blockchain. Up to this document, you have been using a signer to sign operations. For interactive dApps, it is common to use a wallet to sign operations. From a user's perspective, the workflow is as follows:
+
+1. The user has a wallet installed and configured on their device.
+2. The user visits a dApp and wants to interact with it.
+3. The dApp asks the user to connect their wallet.
+4. The user selects their wallet.
+5. The wallet asks the user to confirm the connection.
+6. Now when the dApp wants to send an operation, it asks the wallet to sign it. This might need additional confirmation from the user, or the user might have setup their wallet to sign operations automatically (for instance, automatically sign transactions below a certain amount in a certain time frame).
+
+The main benefit of this workflow is that the user does not have to trust a dApp with their private key. The private key never 
 
 ## Installing the Wallet API
 
 The first thing to do is to use the wallet API is to install it. You just need to install the Taquito package to use the wallet API:
 
 ```
-npm install @taquito/taquito
-
+npm install @taquito/taquito @taquito/beacon-wallet @temple-wallet/dapp
 ```
 
-Once the package is downloaded, you can install the wallet of your choice. The wallet API supports different kinds of wallets. The _Beacon_ and _Temple_ wallets are available to use at the moment. You can install one or both wallets, depending on your requirements:
-
-```
-npm install @taquito/beacon-wallet
-
-npm install @temple-wallet/dapp
-
-```
-
-Remember that some wallets may require an extra step in addition to the package installation. For example, Temple must be used with an extension installed in the browser. We will explain the requirements for the different wallets in detail in the sections below.
+A separate step from setting up the wallet in the dApp code as a developer is to setup the wallet as the user. This step is different for each wallet. Some wallets are browser extensions, while others are mobile apps or web wallets.
```

Comment:
Think this paragraph is trying to explain different wallet packages (beacon/temple) dev can use to setup their walletProvider, found below sentence not relevant with beacon includes access to all extension, mobile and web wallets
> Some wallets are browser extensions, while others are mobile apps or web wallets.

Reply:
Let's say someone who has never tried web3 development is reading the docs. This step suggests that they need to setup a wallet, like what a user of their future dApp needs to do.

Reply:
this context is that `this step(setup wallet) is different for each wallet` with the next sentence saying `Some wallets are browser extensions, while others are mobile apps or web wallets.` will mislead people that for browser, mobile, or web wallets you all need different set up

Reply:
I think Ali's sentence here is fine. Maybe keep the example that Temple needs a browser extension to use properly.

`(e.g. Temple needs the user to install a browser extension)`

Reply:
I understand and will respect if the team think it's fine to keep. 
Just want to be clear I think this paragraph is about to initiate temple instance from temple library (development wallet) is different from initiating beacon instance from beacon library. I find mixing the scope of wallet libraries (temple/beacon) and wallet itself (temple/kukai/plenty/umami) is misleading all diff wallets needs a different set up.

Reply:
@hui-an-yang I see your point. Yeah that is fair.